examples/webgl_morphtargets.html

```diff
@@ -51,96 +51,147 @@
 
 			var camera, scene, renderer;
 
-			var geometry, objects;
-
-			var mesh;
-
 			init();
 			animate();
 
 			function init() {
 
 				container = document.getElementById( 'container' );
 
-				camera = new THREE.PerspectiveCamera( 45, window.innerWidth / window.innerHeight, 1, 15000 );
-				camera.position.z = 500;
+				camera = new THREE.PerspectiveCamera( 45, window.innerWidth / window.innerHeight, 1, 150 );
+				camera.position.z = 10;
 
 				scene = new THREE.Scene();
 				scene.background = new THREE.Color( 0x222222 );
-				scene.fog = new THREE.Fog( 0x000000, 1, 15000 );
-
-				var light = new THREE.PointLight( 0xff2200 );
-				light.position.set( 100, 100, 100 );
-				scene.add( light );
-
-				var light = new THREE.AmbientLight( 0x111111 );
-				scene.add( light );
-
-				var geometry = new THREE.BoxGeometry( 100, 100, 100 );
-				var material = new THREE.MeshLambertMaterial( { color: 0xffffff, morphTargets: true } );
-
-				// construct 8 blend shapes
-
-				for ( var i = 0; i < 8; i ++ ) {
-
-					var vertices = [];
-
-					for ( var v = 0; v < geometry.vertices.length; v ++ ) {
-
-						vertices.push( geometry.vertices[ v ].clone() );
 
-						if ( v === i ) {
-
-							vertices[ vertices.length - 1 ].x *= 2;
-							vertices[ vertices.length - 1 ].y *= 2;
-							vertices[ vertices.length - 1 ].z *= 2;
-
-						}
-
-					}
-
-					geometry.morphTargets.push( { name: "target" + i, vertices: vertices } );
-
-				}
+				var ambientLight = new THREE.AmbientLight( 0x333333 );
+				scene.add( ambientLight );
```

Comment:
You really don't need ambient lighting with the light source at the camera location. I'd remove the ambient light, as the scene is a bit over-bright. (I recognize this is subjective.)